tools/dtrace/README.md

```diff
@@ -1,5 +1,46 @@
 # Oxide DTrace Crucible scripts
 
+## all_downstairs.d
+A DTrace script to show IOs coming and going on all downstairs as well as the
+work task geting new work, performing the work and completing the work. Stats
+are printed at a 4 second interval.
+
+The columns show counts in the last 4 seconds of:
+F>  Flush coming in from the upstairs
+F<  Flush completed message being sent back to the upstairs.
+W>  Write coming in from the upstairs
+W<  Write completed message being sent back to the upstairs.
+R>  Read coming in from the upstairs
+R<  Read completed message being sent back to the upstairs.
+WS  An IO has been submitted to the work task in the downstairs
+WIP An IO is taken off the work queue by the downstairs work task.
+WD  An IO is completed by the downstairs work task.
+
+If a downstairs has not done any IOs it will either print no line, or
+print a line of zeros.
```

Comment:
How would it be that it prints no line? I don't understand from reading the dtrace code how that would happen instead of printing a line of zeroes.

also I think it would be ideal if it always printed a line for each time period if that's possible instead of dropping lines

Reply:
If none of the probes have fired at all, then dtrace won't know about them and will have nothing to print,
but that only happens when the script first starts.  I had a different version of the script that would just
run for 5 seconds and then print a line and exit, and the comment was more applicable there.  I did want
to leave the comment just in case someone ran the script for 10 seconds, hit control-C, they might not
get any output if all the downstairs were idle.

Once any of the downstairs probes in the script has fired, we populate the variable in the script and it
 will print zeros (or new values) for that PID going forward.  It's just we need one IO to prime the pump.

Reply:
> but that only happens when the script first starts ...  It's just we need one IO to prime the pump.

ah perfect, seems fine to me